modules/on_gke/main.tf

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_project_service" "gcr" {
```

Comment:
We typically reserve service configuration to be handled by the Project Factory or a module managed by the user to avoid placing too much responsibility on our modules.

Reply:
Should this go in the user's main.tf?  Like put it in the example main.tf?

Reply:
I put this in the example main.tf's.